packages/typespec-vscode/package.json

```diff
@@ -25,16 +25,12 @@
   "type": "module",
   "main": "./dist/src/extension.cjs",
   "engines": {
-    "vscode": "^1.90.0"
+    "vscode": "^1.91.0"
   },
   "activationEvents": [
     "onLanguage:typespec",
     "onCommand:typespec.restartServer"
   ],
-  "files": [
-    "typespec-vscode-*.vsix",
-    "ThirdPartyNotices.txt"
-  ],
```

Comment:
By removing this, are we including everything now?

Reply:
yeah the vsce tool complained that we had both files and .vscodeignore.
Vscodeignore used to take priority and files was just for the npm package that contains the vsix, so moved to use npmignore for the package

Reply:
moved to this pr as this was was just failing for obscure reason https://github.com/microsoft/typespec/pull/3948